usbtmc/usbtmc.py

```diff
@@ -696,51 +718,77 @@ def clear(self):
             raise UsbtmcException("Clear failed", 'clear')
 
     def remote(self):
-        "Send remote command"
+        """
+        Send the remote command to the instrument.
+        :return:
+        """
         raise NotImplementedError()
 
     def local(self):
-        "Send local command"
+        """
+        Send the local command to the instrument.
+        :return:
+        """
         raise NotImplementedError()
 
     def lock(self):
-        "Send lock command"
-
+        """
+        Advantest/ADCMT ONLY: Send the lock command to the instrument.
```

Comment:
Yeah, that's a bit overkill. And probably in that case I'd argue that the Advantest features are better for something like `python-ivi`. For now this will stay as-is.